rom/dev/doc/error-attribution.md

```diff
@@ -91,23 +91,21 @@ Verifier Library | IMAGE_VERIFIER_ERR_FMC_LOAD_ADDR_INVALID
 Verifier Library | IMAGE_VERIFIER_ERR_FMC_LOAD_ADDR_UNALIGNED                          | 0x000b0022
 Verifier Library | IMAGE_VERIFIER_ERR_FMC_ENTRY_POINT_INVALID                          | 0x000b0023
 Verifier Library | IMAGE_VERIFIER_ERR_FMC_ENTRY_POINT_UNALIGNED                        | 0x000b0024
-Verifier Library | IMAGE_VERIFIER_ERR_FMC_SVN_GREATER_THAN_MAX_SUPPORTED               | 0x000b0025
-Verifier Library | IMAGE_VERIFIER_ERR_FMC_SVN_LESS_THAN_MIN_SUPPORTED                  | 0x000b0026
-Verifier Library | IMAGE_VERIFIER_ERR_FMC_SVN_LESS_THAN_FUSE                           | 0x000b0027
 Verifier Library | IMAGE_VERIFIER_ERR_RUNTIME_LOAD_ADDR_INVALID                        | 0x000b0028
 Verifier Library | IMAGE_VERIFIER_ERR_RUNTIME_LOAD_ADDR_UNALIGNED                      | 0x000b0029
 Verifier Library | IMAGE_VERIFIER_ERR_RUNTIME_ENTRY_POINT_INVALID                      | 0x000b002a
 Verifier Library | IMAGE_VERIFIER_ERR_RUNTIME_ENTRY_POINT_UNALIGNED                    | 0x000b002b
-Verifier Library | IMAGE_VERIFIER_ERR_RUNTIME_SVN_GREATER_THAN_MAX_SUPPORTED           | 0x000b002c
-Verifier Library | IMAGE_VERIFIER_ERR_RUNTIME_SVN_LESS_THAN_MIN_SUPPORTED              | 0x000b002d
-Verifier Library | IMAGE_VERIFIER_ERR_RUNTIME_SVN_LESS_THAN_FUSE                       | 0x000b002e
+Verifier Library | IMAGE_VERIFIER_ERR_FIRMWARE_SVN_GREATER_THAN_MAX_SUPPORTED          | 0x000b002c
```

Comment:
For a separate PR: I would be in favor of just pointing to the source code for this file as well.

Reply:
Agreed. In the meantime though will try and avoid making the problem worse.